packages/text-editor/src/components/StyledTextEditor.svelte

```diff
@@ -15,23 +15,31 @@
 <script lang="ts">
   import { Asset, getResource, IntlString } from '@hcengineering/platform'
   import { getClient } from '@hcengineering/presentation'
-  import { AnySvelteComponent, EmojiPopup, IconEmoji, IconSize, Scroller, showPopup } from '@hcengineering/ui'
+  import {
+    AnySvelteComponent,
+    ButtonSize,
+    EmojiPopup,
+    IconEmoji,
+    IconSize,
+    Scroller,
+    showPopup
+  } from '@hcengineering/ui'
   import { createEventDispatcher } from 'svelte'
   import textEditorPlugin from '../plugin'
   import { RefInputAction, RefInputActionItem, TextEditorHandler, TextFormatCategory } from '../types'
   import Attach from './icons/Attach.svelte'
   import { AnyExtension, mergeAttributes } from '@tiptap/core'
-  import StyleButton from './StyleButton.svelte'
   import TextEditor from './TextEditor.svelte'
   import { Node as ProseMirrorNode } from '@tiptap/pm/model'
+  import Button from '@hcengineering/ui/src/components/Button.svelte'
```

Comment:
Move to import from `@hcengineering/ui`?

Reply:
Updated